action.yml

```diff
@@ -108,7 +108,9 @@ runs:
           '/tmp/imgcmp-stat.json'
         )
 
-        echo "message=${message}" >> $GITHUB_OUTPUT
+        echo "message<<EOF" >> $GITHUB_OUTPUT
+        echo "${message}" >> $GITHUB_OUTPUT
+        echo "EOF" >> $GITHUB_OUTPUT
```

Comment:
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings